### PR TITLE
Added i18n to almost all strings

### PR DESCRIPTION
**Pull request by loviuz**
I have added i18n to all strings, except for terms&conditions and privacy. I have already translated them in italian, so for the other languages the person that will translate Hemmelig should compare the last english version of JSON translation. Do you have plan to move these pages in indipendent files? I think that adding these texts in JSON file for translation couldn't be the right choice.

### CODE REVIEW

**Comment by loviuz** [comment]
I hope this PR will not break anything of important :-)

**Comment by bjarneo** [comment]
Thanks @loviuz. I will have a look at this PR this week. Been busy lately :)

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -2,7 +2,62 @@
...
+            "confirm_password": "Confirm Password",
+            "confirm_new_password": "Confirm your new password",
+            "update_details": "Update details"
+        },
+        "secrets": {
+            "id": "Id",
+            "expires": "Expires",
+            "public": "Public",
+            "delete": "Delete",
+            "yes": "Sì",
```
Comment (bjarneo) [inline public/locales/en/translation.json:42]: maybe "yes" here :)

Comment (loviuz) [reply public/locales/en/translation.json]: fixed :-)

**Comment by bjarneo** [comment]
> Do you have plan to move these pages in indipendent files?

I do not have any plan for anything here :) But, I do agree that it can't be in the same files as the translations.

File: public/locales/it/translation.json [hunk 2]
```diff
@@ -2,7 +2,62 @@
     "sign_in": "Accedi",
     "sign_out": "Esci",
     "sign_up": "Registrati",
-    "account": "Profilo",
```
Comment (bjarneo) [inline public/locales/it/translation.json:5]: A though here. This will most likely break the other translations?

Comment (loviuz) [reply public/locales/it/translation.json:5]: Why? Because these translations doesn't have a "parent"? It works, I have already tried

**Comment by bjarneo** [comment]
You are correct. I do not even remember my own application anymore. Thank you!

**Comment by loviuz** [comment]
Thanks for you great work! :heart:

**Pull request merged**